pkg/storage/chunk/client/congestion/congestion.go

```diff
@@ -17,32 +17,32 @@ func NewController(cfg Config, logger log.Logger, metrics *Metrics) Controller {
 }
 
 func newController(cfg Config, logger log.Logger) Controller {
-	strat := strings.ToLower(cfg.Controller.Strategy)
-	switch strat {
+	start := strings.ToLower(cfg.Controller.Strategy)
```

Comment:
This was not a typo; I don't see how this relates to the topic of the PR?

Reply:
Not sure how that change got in there @dannykopping - will review!

Reply:
Reviewed, and agree this change shouldn't be there. I'm actually mystified as to how it happened. I see a couple of other spelling fixes that I also don't know how happened, but they actually make sense, so keeping them.